modules/ROOT/pages/metrics-list.adoc

```diff
@@ -44,124 +74,141 @@ The last column of the table includes the feature or features that must be enabl
 {empty} +
 
 .Metrics that are available for Open Liberty
-[%header,cols="3,3,6,2,2"]
+[%header,cols="3,3,6,2,2,2"]
 |===
 
 |MicroProfile Metrics 3.0 name
 |MicroProfile Metrics 3.0 Prometheus name(s)
 |Type and description
+|Monitoring component
 |Feature(s) required
 |Version introduced
 
 |classloader.loadedClasses.count
 |base_classloader_loadedClasses_count
 |The number of classes that are currently loaded in the JVM.
 This metric is a gauge.
+|Base metric, no monitoring component associated. Will always be available.
 |{base-metric-features}
 |{mp-2-0}
 
 |classloader.loadedClasses.total
 |base_classloader_loadedClasses_total
 |The total number of classes that were loaded since the JVM started.
 This metric is a counter.
+|Base metric, no monitoring component associated. Will always be available.
 |{base-metric-features}
 |{mp-2-0}
 
 |classloader.unloadedClasses.total
 |base_classloader_unloadedClasses_total
 |The total number of classes that were unloaded since the JVM started.
 This metric is a counter.
+|Base metric, no monitoring component associated. Will always be available.
 |{base-metric-features}
 |{mp-2-0}
 
 |connectionpool.connectionHandles{datasource=<datasourceName>}
 |vendor_connectionpool_connectionHandles{datasource=<dataSourceName>}
 |The number of connections that are in use. This number might include multiple connections that are shared from a single managed connection.
 This metric is a gauge.
+|connectionpool.connectionHandles
```

Comment:
the monitoring components are listed in the Github issue- the component here should be `ConnectionPool`